### PR TITLE
Build a Tribute Page

### PR DESCRIPTION
**Pull request by MarcoDiaz2000**
Objective: Build an app that is functionally similar to https://tribute-page.freecodecamp.rocks/

User Stories:

Your tribute page should have a main element with a corresponding id of main, which contains all other elements.
You should see an element with an id of title, which contains a string (i.e. text), that describes the subject of the tribute page (e.g. "Dr. Norman Borlaug")
You should see either a figure or a div element with an id of img-div
Within the #img-div element, you should see an img element with a corresponding id="image"
Within the #img-div element, you should see an element with a corresponding id="img-caption" that contains textual content describing the image shown in #img-div
You should see an element with a corresponding id="tribute-info", which contains textual content describing the subject of the tribute page
You should see an a element with a corresponding id="tribute-link", which links to an outside site, that contains additional information about the subject of the tribute page. HINT: You must give your element an attribute of target and set it to _blank in order for your link to open in a new tab
Your #image should use max-width and height properties to resize responsively, relative to the width of its parent element, without exceeding its original size
Your img element should be centered within its parent element
Fulfill the user stories and pass all the tests below to complete this project. Give it your own personal style. Happy Coding!

Note: Be sure to add <link rel="stylesheet" href="styles.css"> in your HTML to link your stylesheet and apply your CSS

### CODE REVIEW

**Comment by MarcoDiaz2000** [comment]
Hi @MarcoDiaz2000 

Great job so far!
You still need to work on some issues to go to the next project, but you are almost there!

### Highlights
__________________________________________________________________________________________________________________
- You completed all the app's functionalities 📖


### Required Changes ♻️
__________________________________________________________________________________________________________________
- [ ] You need to install the [linter](https://github.com/microverseinc/linters-config) and run it without errors.
- [ ] Add descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

### **Optional suggestions**
__________________________________________________________________________________________________________________
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@MarcoDiaz2000 ) in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: css/style.css
Comment (MarcoDiaz2000) [inline css/style.css:1]: You need to improve the indentation and remove whitespace. For this project, you can use this [linter](https://github.com/microverseinc/linters-config/tree/master/html-css)

**Review by MarcoDiaz2000 — COMMENTED** [review]
Hi @MarcoDiaz2000

Great job so far!
You still need to work on some issues to go to the next project, but you are almost there!

### Highlights
__________________________________________________________________________________

- You completed all the app's functionalities 📖


### Required Changes ♻️
__________________________________________________________________________________

 

- [ ] You need to install the [linter](https://github.com/microverseinc/linters-config) and run it without errors.

- [ ]  Add descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

### Optional suggestions
__________________________________________________________________________________

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@MarcoDiaz2000 ) in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @MarcoDiaz2000 , 

image

**Congratulations on the good job done so far!** 🤝 

There are some issues that you still need to work on to go to the next project but you are almost there! 😄 

## Highlights ⚡ ⚡ 
- [ ]  Good attempt at making the review 👍🏾 

## Required Changes ♻️
_Check the comments under the review._
- [ ] Good work making the reviews. In this task, you are supposed to use [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) to make the reviews. Please read through the materials and apply the concepts learned to this code review task.
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <!-- <p> Hello World, we are software developers!</p> -->
+    
+    <main id="main">
+      <h1 id="title">Doberman dogs are beatiful!</h1>
+      <div id="img-div">
+        <div id="image-container">
+          <img id="image" src="https://i.blogs.es/1d6dd6/istock-1324413613/840_560.jpeg" alt="">
```
Comment (MarcoDiaz2000) [inline index.html:17]: For SEO it is important to add a name to the images, you can do it by filling in alt="**name of image**"

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+a {
+    text-decoration: none;
+    text-align: center;
+    color: red;
+    width: 100vw;
+    display: flex;
+    justify-content: center;
+    align-items: center;
```
Comment (MarcoDiaz2000) [inline css/style.css:62]: Expected indentation of 2 spaces

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+
+
+
+
+#tribute-info {
+    justify-content: center;
+    text-align: justify;
+    padding: 0 100px;
```
Comment (MarcoDiaz2000) [inline css/style.css:51]: Expected indentation of 2 spaces

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+}
+h1, #img-caption {
+    width: 100vw;
+    justify-content: center;
+    text-align: center;
+    font-size: 3em;
+}
+
+#img-caption {
+    font-size: 1.5em;
```
Comment (MarcoDiaz2000) [inline css/style.css:41]: Expected indentation of 2 spaces

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
+h1, #img-caption {
+    width: 100vw;
+    justify-content: center;
+    text-align: center;
+    font-size: 3em;
```
Comment (MarcoDiaz2000) [inline css/style.css:37]: Expected indentation of 2 spaces

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+#image {
+    max-width: 100%; 
+    height: auto;
+}
+#main {
+    width: 100vw;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
```
Comment (MarcoDiaz2000) [inline css/style.css:31]: Expected indentation of 2 spaces

File: css/style.css [hunk 7]
```diff
@@ -0,0 +1,64 @@
+
+
+:root {
+    --blue: rgb(92, 92, 207);
+    --background: rgb(214, 211, 232);
```
Comment (MarcoDiaz2000) [inline css/style.css:5]: Expected indentation of 2 spaces

File: css/style.css [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+
+
```
Comment (MarcoDiaz2000) [inline css/style.css:9]: Preferably just leave a blank line after each rule

File: css/style.css [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+* {
+    background-color: var(--background);
```
Comment (MarcoDiaz2000) [inline css/style.css:11]: Expected indentation of 2 spaces

File: css/style.css [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+#image-container {
+    display: flex;
+    width: 100vw;
+    height: 100vh;
+    justify-content: center;
+    align-items: center;
```
Comment (MarcoDiaz2000) [inline css/style.css:19]: Expected indentation of 2 spaces

File: css/style.css [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+#image {
+    max-width: 100%; 
+    height: auto;
```
Comment (MarcoDiaz2000) [inline css/style.css:24]: Expected indentation of 2 spaces

**Review by MarcoDiaz2000 — COMMENTED** [review]
Hi @MarcoDiaz2000

### Great job so far!
-----------------------------------------------------------------------------
You still need to work on some issues to go to the next project, but you are almost there!



### Highlights
-----------------------------------------------------------------------------
- You completed all the app's functionalities 📖

### Required Changes ♻️
-----------------------------------------------------------------------------
- [ ] You need to install the [linter](https://github.com/microverseinc/linters-config) and run it without errors.
- [ ] Add descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.
- [ ] Kindly check the inline comments under the review. ⬇️

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@MarcoDiaz2000 ) in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: css/style.css (hunk 2 above)
Comment (GedeonTS) [reply css/style.css:62]: - [REQUIRED] Good job using inline comments as requested by the previous reviewer :clap: , However, for this comment and all the comments below, kindly make sure to make the comment clear, adding also the purpose of the change :recycle: , as it is a learning opportunity for the student :recycle:

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :
Hi @MarcoDiaz2000 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights :tophat: :+1: 

- Good use of GitHub review feature :+1: 
- Good use of inline comments :clap: 
 ## Required Changes ♻️
_Check the comments under your comments._

## Optional suggestions :ringed_planet: 
- [OPTIONAL] I suggest using more emojis, to make your comments more human-friendly:recycle: 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (MarcoDiaz2000) [inline index.html:17]: For SEO  it is important to add a name to the images ⚡you can do it by filling in alt="name of image" 👍🏾

File: css/style.css (hunk 2 above)
Comment (MarcoDiaz2000) [inline css/style.css:62]: For best practices ♻️ in HTML and CSS, it is expected that an indentation of 2 spaces will be added ⚡

File: css/style.css (hunk 3 above)
Comment (MarcoDiaz2000) [inline css/style.css:51]: For best practices ♻️ in HTML and CSS, it is expected that an indentation of 2 spaces will be added ⚡

File: css/style.css (hunk 4 above)
Comment (MarcoDiaz2000) [inline css/style.css:41]: For best practices ♻️ in HTML and CSS, it is expected that an indentation of 2 spaces will be added ⚡

File: css/style.css (hunk 5 above)
Comment (MarcoDiaz2000) [inline css/style.css:37]: For best practices ♻️ in HTML and CSS, it is expected that an indentation of 2 spaces will be added ⚡

File: css/style.css (hunk 6 above)
Comment (MarcoDiaz2000) [inline css/style.css:31]: For best practices ♻️ in HTML and CSS, it is expected that an indentation of 2 spaces will be added ⚡

File: css/style.css (hunk 11 above)
Comment (MarcoDiaz2000) [inline css/style.css:24]: For best practices ♻️ in HTML and CSS, it is expected that an indentation of 2 spaces will be added ⚡

File: css/style.css (hunk 10 above)
Comment (MarcoDiaz2000) [inline css/style.css:19]: For best practices ♻️ in HTML and CSS, it is expected that an indentation of 2 spaces will be added ⚡

File: css/style.css (hunk 9 above)
Comment (MarcoDiaz2000) [inline css/style.css:11]: For best practices ♻️ in HTML and CSS, it is expected that an indentation of 2 spaces will be added ⚡

File: css/style.css (hunk 7 above)
Comment (MarcoDiaz2000) [inline css/style.css:5]: For best practices ♻️ in HTML and CSS, it is expected that an indentation of 2 spaces will be added ⚡

**Review by MarcoDiaz2000 — COMMENTED** [review]
Hi @MarcoDiaz2000

Great job so far! 👏👏👏
------------------------------------------------------------------------
You still need to work on some issues to go to the next project, but you are almost there!




### Highlights 🎩 👍
------------------------------------------------------------------------
You completed all the app's functionalities 📖

### Required Changes ♻️
------------------------------------------------------------------------

- [ ] You need to install the [linter](https://github.com/microverseinc/linters-config) and run it without errors.	 ♻️

- [ ] Add descriptive README file to your project - please use [this template] 👍🏾(https://github.com/microverseinc/readme-template) but remember to customize it to your project. 📖

- [ ] Kindly check the inline comments under the review. ⬇️

### Optional suggestions  🪐
------------------------------------------------------------------------
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@MarcoDiaz2000 ) in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by joel-kalema — APPROVED** [review]
Hi @MarcoDiaz2000 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Your comments are clear and easy to understand 💯 ✔️
You always include an explanation ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@joel-kalema** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**